command/rootchain/emit/emit.go

```diff
@@ -59,6 +59,13 @@ func setFlags(cmd *cobra.Command) {
 		nil,
 		"list of amounts to fund wallets",
 	)
+
+	cmd.Flags().StringVar(
+		&params.jsonRPCAddress,
+		jsonRPCFlag,
+		"",
```

Comment:
Default to localhost?

Reply:
Fixed https://github.com/0xPolygon/polygon-edge/pull/989/commits/3732f6e0b662b6ace6f9c297999c9ff6ffc5efd1